From 9674cfc8e54f7cac7a890e4e54bed001cc897192 Mon Sep 17 00:00:00 2001
From: imsi32 <81227251+imsi32@users.noreply.github.com>
Date: Wed, 20 Nov 2024 00:03:51 +0200
Subject: [PATCH 03/20] [FIX] Blog Page Navigation (#2841) (#2846)

I tested on my Github page and it worked correctly. Then, copied here.

Issue: Blog page navigation extends layout of the page in small screens.
Fixes: #2841
---
 _includes/pagination.liquid | 4 ++--
 _sass/_base.scss            | 1 +
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/_includes/pagination.liquid b/_includes/pagination.liquid
index a8a2961f..61bf4fb6 100644
--- a/_includes/pagination.liquid
+++ b/_includes/pagination.liquid
@@ -3,7 +3,7 @@
     <ul class="pagination pagination-lg justify-content-center">
       <li class="page-item {% unless paginator.previous_page %}disabled{% endunless %}">
         <a class="page-link" href="{{ paginator.previous_page_path | relative_url }}" tabindex="-1" aria-disabled="{{ paginator.previous_page }}"
-          >Newer</a
+          >&lt;</a
         >
       </li>
       {% if paginator.page_trail %}
@@ -14,7 +14,7 @@
         {% endfor %}
       {% endif %}
       <li class="page-item {% unless paginator.next_page %}disabled{% endunless %}">
-        <a class="page-link" href="{{ paginator.next_page_path | relative_url }}">Older</a>
+        <a class="page-link" href="{{ paginator.next_page_path | relative_url }}">&gt;</a>
       </li>
     </ul>
   </nav>
diff --git a/_sass/_base.scss b/_sass/_base.scss
index d226dca9..f9840f80 100644
--- a/_sass/_base.scss
+++ b/_sass/_base.scss
@@ -589,6 +589,7 @@ footer.sticky-bottom {
   .page-item {
     .page-link {
       color: var(--global-text-color);
+      padding: 0.75rem 1.15rem;
 
       &:hover {
         color: $black-color;
-- 
2.48.1

